types/externs.d.ts

```diff
@@ -323,6 +323,7 @@ declare global {
       };
       pid: number;
       tid: number;
+      /* Microseconds. */
```

Comment:
```suggestion
      /** Timestamp of the event in microseconds. */
```

double `**` the important bit for jsdoc intellisense I believe?